### PR TITLE
Prevent Blackberry screen sleeping. CCScrollView TouchPriority

### PR DESCRIPTION
**Pull request by edbartley**
Added code to prevent the Blackberry screen from going to sleep while the game is in the foreground.

The CCScrollView touchPriority should come from the CCLayer, not fixed to 0. This allows the developer to modify the touch priority of the scrollview.

### CODE REVIEW

File: cocos2dx/platform/blackberry/CCEGLView.cpp [hunk 1]
```diff
@@ -391,6 +391,13 @@ bool CCEGLView::initGL()
         return false;
     }
 
+    int idle_mode = SCREEN_IDLE_MODE_KEEP_AWAKE;
+    rc = screen_set_window_property_iv(m_screenWindow, SCREEN_PROPERTY_IDLE_MODE, &idle_mode);
+    if (rc)
+    {
+        perror("screen_set_window_property_iv(SCREEN_PROPERTY_IDLE_MODE)");
+        return false;
+    }
```
Comment (dumganhar) [inline cocos2dx/platform/blackberry/CCEGLView.cpp:400]: Do we need to provide an interface for developer to support this functionality or not?

Comment (edbartley) [inline cocos2dx/platform/blackberry/CCEGLView.cpp:400]: I was thinking about that. Yes, that is a good idea.

Comment (dumganhar) [inline cocos2dx/platform/blackberry/CCEGLView.cpp:400]: Hi @edbartley ,
Do you mind to tell us when will this Pull Request be updated? Thanks.

Comment (edbartley) [inline cocos2dx/platform/blackberry/CCEGLView.cpp:400]: @dumganhar I didn't realize you wanted me to do more with this pull request. I guess you want me to add the interface. I can do that but it'll be a little while, maybe the middle of next month. Please close this pull request if it's not acceptable as-is and I'll send a new pull request with a proper interface.

Is the "CCScrollView TouchPriority" code acceptable? If so, can you merge that specific code? If not, I'll send a new pull request with just that code.

Cheers!

**Comment by edbartley** [comment]
Closing until I can make the necessary changes.

**Pull request closed without merging**